src/adv.py

```diff
@@ -33,11 +35,59 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+
+inits = 'Start of Game'
+location = 'outside'
+
 #
 # Main
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player('Kevin', room['outside'])
+current_room = player.current_room
+
+#============== FUNCTIONS ====================#
+
+
+# moves player to user given direction
+def move_area(p_inpt):
+    global current_room
+    if p_inpt == 'n':
+        if hasattr(current_room, "n_to"):
+            new_room = current_room.n_to
+            current_room = new_room
+            print(
+                f"You are now at the {current_room.name}. {current_room.description}.")
+            if len(current_room.items) > 0:
+                print(f"You see a {current_room.items[0]}.")
```

Comment:
You can put a room inventory printing method on your Room class to print out everything you see.

Reply:
Instead of just the first one.